dev-python/python-jose/python-jose-3.2.0-r1.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2020 Gentoo Authors
```

Comment:
If you don't mind some extra work, please switch to `epytest --deselect` here.

Reply:
Have been done gladly. I collect all your comments to make every PR better than the previous one (and to fix various python ebuilds found in tree).

I did it using a full array (similar to how it was done at `dev-python/virtualenv`) so it will be better ready to future needs.